.github/workflows/github-actions-ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       - name: run test in docker
         run: |
           docker run \
-              --env-file /home/github_worker/env.list \
+              --env-file ~/env.list \
```

Comment:
just curious why this needed to be changed? Due to moving VM to platform9?

Reply:
Yes, this change makes it compatible with both workers. The current one has /home/github_worker while the platform 9 VM users /local1/github_worker. "~/" maps to either.